File: support_tools/swigwin-3.0.12/Examples/test-suite/python/li_implicit_runme.py
from li_implicit import *
b = B()
ai = A(1)
ad = A(2.0)
ab = A(b)

ai, get(ai)
ad, get(ad)
ab, get(ab)

if get(ai) != get(1):
    raise RuntimeError, "bad implicit type"
if get(ad) != get(2.0):
    raise RuntimeError, "bad implicit type"
if get(ab) != get(b):
    raise RuntimeError, "bad implicit type"

File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/li_implicit_runme.m
# do not dump Octave core
if exist("crash_dumps_octave_core", "builtin")
  crash_dumps_octave_core(0);
endif

li_implicit
b = B();
ai = A(1);
ad = A(2.0);
ab = A(b);

ai, get(ai);
ad, get(ad);
ab, get(ab);

if (get(ai) != get(1))
  error("bad implicit type")
endif
if (get(ad) != get(2.0))
  error("bad implicit type")
endif
if (get(ab) != get(b))
  error("bad implicit type")
endif

